checkstyle/checkstyle-suppressions.xml

```diff
@@ -22,6 +22,8 @@
     "https://checkstyle.org/dtds/suppressions_1_2.dtd">
 <suppressions>
     <suppress checks="EmptyLineSeparator" files="BCrypt\.java|ConcurrentLinkedDeque8\.java"/>
-
     <suppress checks="EmptyLineSeparator" files="XGBoostModelLexer\.java|XGBoostModelParser\.java"/>
+
+    <suppress checks="ConstantName" files="BCrypt\.java|ConcurrentLinkedDeque8\.java|PlatformCallbackOp\.java"/>
+    <suppress checks="ConstantName" files="XGBoostModelLexer\.java|XGBoostModelParser\.java"/>
```

Comment:
Can be combined with `EmptyLineSeparator`: 'checks="EmptyLineSeparator|ConstantName"`